charts/aws-ebs-csi-driver/CHANGELOG.md

```diff
@@ -1,5 +1,8 @@
 # Helm chart
 
+## v2.22.1
+Feature flag to disable daemonset creation for linux  ([#1732](https://github.com/kubernetes-sigs/aws-ebs-csi-driver/pull/1732), [@monicastanciu](https://github.com/monicastanciu))
+
```

Comment:
Thanks for adding this : ) 

We typically have a maintainer who performs releases make these changes so let's remove it.